core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1529,7 +1529,10 @@ public MetricKey build() {
           .build();
   public static final MetricKey FUSE_CACHED_PATH_COUNT =
       new Builder("Fuse.CachedPathCount")
-          .setDescription("Total number of Alluxio paths to cache for FUSE conversion.")
+          .setDescription(String
+              .format("Total number of Alluxio paths to cache for FUSE conversion. "
+                      + "This value will be smaller or equal to %s",
```

Comment:
Maybe use "must be smaller than" instead of "will be smaller"?

Reply:
If it's a property key, then must be smaller is better, since it guides how users how to set the property key.
For metric key, it's a value show to users. So description only reflects what will happen instead of requirements.